deploy/controller_config.yaml

```diff
@@ -14,3 +14,4 @@ data:
   devworkspace.default_routing_class: "basic"
   # image pull policy that is applied to all workspace's containers
   devworkspace.sidecar.image_pull_policy: Always
+  devworkspace.api_sidecar.image: quay.io/che-incubator/che-api-sidecar:next
```

Comment:
++++1, I always forget to add it when deploying :)

Reply:
+1 as well, I kept having to add this when I was testing the kubernetes cert gen pr